### ISSUE TITLE
Feature Request: Command to save history rather than interval

### ISSUE THREAD

**Issue by conner-fallone** [gayanhewa/vscode-localhistory#2]
It would be nice to be able to use a command to save local history, rather than relying on an interval. This would almost act like source control where I could save history manually, and maybe only save the initial history of the file if that is the only file i care to diff against.

**Comment by tatsujb** [comment]
yes! I support this. actually it should be a combination of save and time.

Saving is the trigger but no new triggers are acknowledged until a minute has gone by, or better yet : 

create a history state for every version of the file succeeded by more than a minute of not being edited.

this way would be so clever and intuitive.
